k8s/templates/vineyardd/deployment.yaml

```diff
@@ -167,19 +142,15 @@ spec:
           volumeMounts:
           - name: log
             mountPath: /var/log/vineyard
-        {{- end }}   
+        {{- end }}
       volumes:
       - name: run
         hostPath:
-          {{- if .Spec.Config.Socket }}
-          path: {{ .Spec.Config.Socket }}
-          {{ else }}
-          path: /var/run/vineyard-{{ .Namespace }}-{{ .Name }}
-          {{- end }}
-      {{- if .Spec.Config.SpillConfig.Name }}
-      - name: {{ .Spec.Config.SpillConfig.Name }}
+          path: {{ .Spec.Socket }}
```

Comment:
I think the e2e test failed at this, it should be `/var/run`.

Reply:
And all path `/var/run/vineyard-vineyard-system-vineyardd-sample` should be changed to the default socket path `/var/run`. https://github.com/v6d-io/v6d/blob/71c0afedef8aceffbf3bd1ed3db390741ba689a0/k8s/test/e2e/workflow-job1.yaml#L50

FYI, @sighingnow